src/control/CMakeLists.txt

```diff
@@ -10,8 +10,11 @@ set(IMPLEMENTATION_FILES
   src/pid/pid.cpp
   src/adapter_control/adapter.cpp
   src/adapter_control/fsds.cpp
+  src/pure_pursuit/pp.cpp
 )
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../common_lib/include)
```

Comment:
This is not how I was thinking of including the other directory, but it might not be a bad idea